src/main/pg/pinio.c

```diff
@@ -63,10 +91,18 @@ void pgResetFn_pinioConfig(pinioConfig_t *config)
     config->ioTag[1] = IO_TAG(PINIO2_PIN);
     config->ioTag[2] = IO_TAG(PINIO3_PIN);
     config->ioTag[3] = IO_TAG(PINIO4_PIN);
+    config->ioTag[4] = IO_TAG(PINIO5_PIN);
+    config->ioTag[5] = IO_TAG(PINIO6_PIN);
+    config->ioTag[6] = IO_TAG(PINIO7_PIN);
+    config->ioTag[7] = IO_TAG(PINIO8_PIN);
```

Comment:
Maybe support only 4 or 8, avoiding too many #ifs:
```suggestion
#if PINIO_COUNT > 4
    config->ioTag[4] = IO_TAG(PINIO5_PIN);
    config->ioTag[5] = IO_TAG(PINIO6_PIN);
    config->ioTag[6] = IO_TAG(PINIO7_PIN);
    config->ioTag[7] = IO_TAG(PINIO8_PIN);
#endif
```

Reply:
this does make sense though, use either 4 or 8, and specifically because of the pinio boxes. If we didn't need a box for every pin, then the code as it was in my initial PR I feel could have been fine: 4 boxes, 4 pins, then the pinio array being exactly sized up to 8.

Reply:
Using >4 was mainly to reduce number on #if. IMO enforcing 4 or 8 should fit most use cases.
But it is possible to guard each `ioTag` assingment
(or, better, use template instead of function)